### ISSUE TITLE
[function] array_repeat

### ISSUE THREAD

**Issue by wangsimo0** [StarRocks/starrocks#19658]
- [x] https://github.com/StarRocks/starrocks/pull/47862
need a function to repeatedly generate array.

## description

```
array_repeat (element, count)
```
in which,
element: can be any data type that supported in an array, including array.
count: integer

## examples

```
select array_repeat(1,5) ;
[1,1,1,1,1]

select  array_repeat([1,2],3);
[[1,2],[1,2],[1,2]]

```

**Comment by lqkkkkk** [comment]
Hi @kateshaowanjou , can you assign this array_repeat  to me？ At this time I will definitely make it, Thank you. best wishes：）

**Comment by wangxiaobaidu11** [comment]
hi I am beike olap developer. Can you assign me ?

**Comment by kateshaowanjou** [comment]
@wangxiaobaidu11 This issue is assigned to you. Have fun coding!

**Comment by kateshaowanjou** [comment]
@wangxiaobaidu11 sorry, someone before you already asked for this issue. Can you take another one? ❤️

**Comment by liangjiawei1110** [comment]
I'd like to try it.

**Comment by kateshaowanjou** [comment]
@liangjiawei1110 Hi there, are you still working on this issue? If you need any help, you can add my wechat:wanjoushao. Thx.

**Comment by kimmking** [comment]
@kateshaowanjou May I pick this task?

**Comment by wangsimo0** [comment]
@kimmking of course, thank you so much. also this function will help sr's trino's compatibility. when you developing this function, could you plz also consider the convert? I think it can be fixed in Trino2SRFunctionCallTransformer

**Comment by chenminghua8** [comment]
@kimmking Can you assign this task to me?

**Comment by kateshaowanjou** [comment]
> @kimmking Can you assign this task to me?

I'm glad to see that you are willing to work on it. I've assigned this task to you since it was taken a long time ago by kimmking and we have not yet seen a PR, I suppose he's not working on it.

**Comment by chenminghua8** [comment]
OK, I'm developing this feature.

**Comment by chenminghua8** [comment]
Why can't the RP I submitted execute the test process?

**Comment by chenminghua8** [comment]
Why don't code owners do code reviews?
